lib/ddtrace/context.rb

```diff
@@ -133,12 +133,14 @@ def sampled?
     # can be re-used immediately.
     #
     # This operation is thread-safe.
-    def get
+    def get(priority_sampling = false)
```

Comment:
I turned this into an argument because it would be tricky to get the value of `priority_sampling` from the `Tracer` on every instantiation of `Context` objects. If you have any other ideas, let me know.